apps/password-reset/src/app/services/password-reset-config.service.ts

```diff
@@ -14,15 +14,29 @@ export class PasswordResetConfigService {
     private appConfigService: AppConfigService,
     private storeService: StoreService,
     private location: Location,
-    private authzSevice: AuthzResolverService,
-    private guiAuthResolver: GuiAuthResolver
+    private authzSevice: AuthzResolverService
   ) { }
 
   loadConfigs(): Promise<void> {
     return this.appConfigService.loadAppDefaultConfig()
       .then(() => this.appConfigService.loadAppInstanceConfig())
       .then(() => this.setApiUrl())
-      .then(() => this.appConfigService.setInstanceFavicon());
+      .then(() => this.appConfigService.setInstanceFavicon())
+      .then(() => this.initAuthService.verifyAuth())
```

Comment:
the authentication should happen only and only if in the url ISNT "token" parameter